view.php

```diff
@@ -407,30 +477,52 @@
             $strenc = ($zoom->option_encryption_type === ZOOM_ENCRYPTION_TYPE_E2EE)
                 ? $strencryptionendtoend
                 : $strencryptionenhanced;
-            $table->data[] = [$strencryption, $strenc];
+            $rowencryption = new html_table_row();
+            $rowencryption->id = 'security-row-encryption';
+            $thencryption = new html_table_cell($strencryption);
+            $thencryption->header = true;
+            $rowencryption->cells = [$thencryption, $strenc];
+            $table->data[] = $rowencryption;
         }
     }
 
     // Show waiting room.
     if (!$zoom->webinar) {
         $strwr = ($zoom->option_waiting_room) ? $stryes : $strno;
-        $table->data[] = [$strwwaitingroom, $strwr];
+        $rowwaitingroom = new html_table_row();
+        $rowwaitingroom->id = 'security-row-waitingroom';
+        $thwaitingroom = new html_table_cell($strwwaitingroom);
+        $thwaitingroom->header = true;
+        $rowwaitingroom->cells = [$thwaitingroom, $strwr];
+        $table->data[] = $rowwaitingroom;
     }
 
     // Show join before host.
     if (!$zoom->webinar) {
         $strjbh = ($zoom->option_jbh) ? $stryes : $strno;
-        $table->data[] = [$strjoinbeforehost, $strjbh];
+        $rowjoinbeforehost = new html_table_row();
+        $rowjoinbeforehost->id = 'security-row-joinbeforehost';
+        $thjoinbeforehost = new html_table_cell($strjoinbeforehost);
+        $thjoinbeforehost->header = true;
+        $rowjoinbeforehost->cells = [$thjoinbeforehost, $strjbh];
+        $table->data[] = $rowjoinbeforehost;
     }
 
     // Show authentication.
-    $table->data[] = [$strauthenticatedusers, ($zoom->option_authenticated_users) ? $stryes : $strno];
+    $rowauthenticatedusers = new html_table_row();
+    $rowauthenticatedusers->id = 'security-row-authenticatedusers';
+    $thauthenticatedusers = new html_table_cell($strauthenticatedusers);
+    $thauthenticatedusers->header = true;
+    $rowauthenticatedusers->cells = [$thauthenticatedusers, $zoom->option_authenticated_users];
```

Comment:
Seems like it's going to show "1" or "0". The original behavior would probably be:
```suggestion
    $rowauthenticatedusers->cells = [$thauthenticatedusers, $zoom->option_authenticated_users ? $stryes : $strno];
```

The reason this one is different from some of the others is that the Yes/No value is saved into a variable.
```suggestion
    $strauthenticatedusers = $zoom->option_authenticated_users ? $stryes : $strno;
    $rowauthenticatedusers->cells = [$thauthenticatedusers, $strauthenticatedusers];
```